tests/macsec/test_macsec.py

```diff
@@ -205,3 +245,122 @@ def check_mka_establishment():
         disable_macsec_port(duthost, port_name)
         disable_macsec_port(nbr["host"], nbr["port"])
         delete_macsec_profile(nbr["host"], profile_name)
+
+def get_lldp_list(host):
```

Comment:
This function should be moved to macsec_platform_helper.py

Reply:
Fix it